src/main/java/uk/gov/hmcts/reform/professionalapi/domain/AccessType.java

```diff
@@ -25,4 +25,15 @@ public AccessType(@JsonProperty(value = "jurisdictionId") String jurisdictionId,
         this.enabled = enabled;
     }
 
+    public static AccessType fromUserConfiguredAccess(UserConfiguredAccess userConfiguredAccess) {

```

Comment:
I think ive removed this kinda method from the domain class, because its only needed by the test. Better to use it in one of test classes - RefDataUtil

Reply:
addressed in commit [8ea5e96](https://github.com/hmcts/rd-professional-api/pull/1529/commits/8ea5e969fad63b555ae4f3f8928ccfa957f73b68) - using method in RefDataUtil class